src/modules/generators/cylinders.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2015 The Noise-rs Developers.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use num::{Float, One};
+use math;
+use math::Point3;
+
+use Module;
+
+pub struct Cylinders<T: Float> {
+    frequency: T,
+}
```

Comment:
I was intending for the fields to remain private so that the modules would stay immutable once generated, preventing the use of mut Module to modify the internal field values.